src/cli/create-component.ts

```diff
@@ -7,12 +7,18 @@ const createFile = (folderPath: string, filename: string): void => fse.ensureFil
 const dashPrefix = (prefix?: string) => prefix === undefined ? '' : `${prefix}-`;
 
 const createDocumentation = (componentDirectory: string, component: string): void => {
-  const content = `## ${component}\n`;
+  const content = `## ${createDocumentationTitle(component)}\n`;
   const file = `${component}.md`;
   createFile(componentDirectory, file);
   fs.writeFileSync(path.resolve(componentDirectory, file), content);
 };
 
+const createDocumentationTitle = (component: string): string => {
+  const componentNameWithoutDash = component.replace(/-/g, ' ');
```

Comment:
maybe try `component.split('-').join(' ')` to not use a Regex ?

Reply:
I feel that you have a problem with regexes. I'll make the change.

Reply:
Done